seed/seedwriter/writer.go

```diff
@@ -1258,7 +1347,7 @@ func (w *Writer) WriteMeta() error {
 // query accessors
 
 func (w *Writer) checkSnapsAccessor() error {
-	if w.expectedStep < seedSnapsStep {
+	if !w.checkStepCompleted(downloadedStep) {
```

Comment:
This used to be `seedSnapsStep` and is now `downloadedStep` - is this intentional?

Reply:
Good question! And yes, it is intentional, the logic is now checking whether a step is completed, not if the current step is before. Same result, but a bit more readable